modules/@apostrophecms/modal/ui/apos/components/AposModalShareDraft.vue

```diff
@@ -175,15 +175,19 @@ export default {
       });
     },
     generateShareUrl(aposShareKey) {
-      const url = new URL(`${location.origin}${this.doc._url}`);
+      const url = new URL(this.doc._url);
```

Comment:
In my tests `this.doc._url` did not contain the origin, therefore `new URL()` was failing.
It was with a article

Reply:
It might or might not, it depends on whether baseUrl is present. So you should check and if you don't see it starts with a protocol you could reasonably do this as a workaround but otherwise it would cause a doubled prefix situation.